doc/developer_guide.md

```diff
@@ -1,3 +1,62 @@
+# Environment setup
+
+You need Python >= 3.6 installed, and you need [tox](https://pypi.org/project/tox/) >= 3.10.
+
+Using tox takes care of all the details of creating and managing an isolated
+virtual environment, installing the necessary dependencies, and isolating
+testsuite runs.
+
+The command for using tox is: `tox {-e envlist} {-- arguments}`. Use `tox -a
+-v` to get a list of available environments.
+
```

Comment:
Maybe right here it would be helpful to just state something like

> To set up and activate the tox environment for Plover and install all dependencies, in your cloned repository simply run:
>
> `tox`

It's so easy that I almost didn't trust it.

Reply:
Alternatively I would put this right at the top, after saying that python and tox are required.

That way, I could see someone who's just trying to run from source would figure it out…

- Install python
- pip install tox
- tox
- Mac and Linux: ./launch.sh
- Windows: ./launch.bat

Or maybe we just need a really brief "Running from Source" for someone who's just looking to get set up as quickly as possible.

Reply:
Except that won't work, you need to activate the environment before using lanch.sh/launch.bat. Honestly, I almost remove those. And whether you're using the bat or the sh on Windows will depend on your environment: since we requiring bash to be present for creating the distribution, someone might use the git shell.

Reply:
Fair, so would the real bootstraps be

- Install python
- pip3 install tox
- tox -e launch

?

Reply:
I'm also cool with removing the launch scripts 👍

Reply:
OK, I'll remove the launch/test scripts.